### PR TITLE
Remove Saltpack prefixes from some consts and vars

### PR DESCRIPTION
**Pull request by akalin-keybase**
This is the recommended Go style.

Also convert some exported vars to functions to avoid
modifications.

Fix a few typos and remove a few extraneous newlines.

Add a TODO for checking FormatName.

### CODE REVIEW

**Comment by akalin-keybase** [comment]
This is pulled out of https://github.com/keybase/saltpack/pull/18 .

File: const.go [hunk 1]
```diff
@@ -28,9 +28,16 @@ const MessageTypeDetachedSignature MessageType = 2
 // signcrypted message.
 const MessageTypeSigncryption MessageType = 3
 
-var SaltpackVersion1 = Version{Major: 1, Minor: 0}
-var SaltpackVersion2 = Version{Major: 2, Minor: 0}
-var SaltpackCurrentVersion = SaltpackVersion1
+func Version1() Version {
```
Comment (akalin-keybase) [inline const.go]: One change that wasn't in #18: changing these to functions.

Comment (oconnor663) [inline const.go]: Any particular reason? (I'm fine either way, just curious.)

Comment (akalin-keybase) [inline const.go]: If they're vars, that means that they're modifiable, which I don't think we want.

**Pull request merged**